src/ngraph/runtime/cpu/pass/cpu_fusion.hpp

```diff
@@ -35,21 +35,40 @@ namespace ngraph
 class ngraph::runtime::cpu::pass::CPUFusion : public ngraph::pass::GraphRewrite
 {
 public:
-    CPUFusion()
+    //30 different fusion groups
+    //that we can nest/mix&match/etc
+    //should be good enough
+    //for quite a while
+    enum fusions
+    {
+        DIFFERENTIABLE_FUSIONS = 0x1,
```

Comment:
Add a comment "fusions that result in ops that are differentiable through ngraph's autodiff"